Squidex/squidex
# Feature/identity server public origing to base url

This PR sets the publicoriging of IdentityServer to the configured Baseurl.

How to test:
- Pull code
- Build code
- Go to: http://localhost:5000/identity-server/.well-known/openid-configuration
- Confirm that the provided URL's start with the configuered BaseUrl
- Change the BaseUrl in the appsettings
- Run again. And confirm that the new BaseUrl is displayed in the openid-configuration file
- Confirm that clicking the login button opens a popup with the url of the newly configured BaseUrl
